runtime/JavaScript/package.json

```diff
@@ -42,7 +42,8 @@
   },
   "scripts": {
     "build": "webpack",
-    "test": "jasmine",
+    "test": "jasmine && npm run test:projects",
```

Comment:
can we rename `projects` to `builds` ?

Reply:
do you also want me to rename the directory "setups" to "builds" ?

Reply:
Yes that would make sense